test/system/030-run.bats

```diff
@@ -1382,4 +1382,14 @@ search               | $IMAGE           |
     run_podman container rm $cname
 }
 
+# https://issues.redhat.com/browse/RHEL-14469
+@test "podman run - /run must not be world-writable in systemd containers" {
+    run_podman run -d --rm $SYSTEMD_IMAGE /usr/sbin/init
+    cid=$output
+    run_podman exec -it $cid stat -c '%a' /run
+    assert "$output" =~ "755" "stat /run"
```

Comment:
*equals*, not equal-tilde please.

Reply:
Nope, the output of "stat -c '%a' /run" is '755/r' or '1777/r' on the RHEL 8.y.

Reply:
Use `exec`, not `exec -it`. `-it` is almost always a bad idea.

Reply:
Done.